app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -42,3 +48,55 @@ body {
     color: $light-forest-green;
     text-align: left;
  }
+.fw-normal {
+    font-weight: normal;
+}
+.inline {
+    display: inline;
+}
+.f-left {
+    float: left;
+}
+.f-right {
+    float: right;
+}
+.label-tag {
+    line-height: 1.1;
+    background-color: $cyan;
+    display: inline-block;
+    max-width: 100%;
+    margin: 4px 4px 0px 0px;
+    font-size: 83%;
+    font-weight: bold;
+    color: $white;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: .25em;
+    padding: 4px;
+}
+a.label-tag:hover,
+a.label-tag:focus {
+  color: $white;
+  text-decoration: none;
+  cursor: pointer;
+}
+.well {
+    min-height: 10px;
+    padding: 0px 0px 0px 3px;
+    margin-bottom: 0px;
+    margin-top: 20px;
+}
+a {
+  color: $gray;
+  text-decoration: none;
+  -webkit-transition: all 0.5s ease;
+  -moz-transition: all 0.5s ease;
+  -o-transition: all 0.5s ease;
+  transition: all 0.5s ease;
+}
+a:hover,
+a:focus {
+  color: $pink;
+  text-decoration: none;
+}
```

Comment:
since this is sass, can be:
```
a {
  color: $gray;
  text-decoration: none;
  -webkit-transition: all 0.5s ease;
  -moz-transition: all 0.5s ease;
  -o-transition: all 0.5s ease;
  transition: all 0.5s ease;

  &:hover,
  &:focus {
    color: $pink;
    text-decoration: none;
  }
}